### DISCUSSION TITLE
poll the status of HTTP

### DISCUSSION THREAD

**Discussion by myluluy** [langgenius/dify#6669] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

How can I use a node in the workflow to poll the status of an interface, such as a taskStatus interface? The iterator node doesn't seem to be suitable

**Comment by crazywoola** [comment]
We do not support this feature right now.

**Comment by myluluy** [comment]
Will it be supported in future versions？
